db/migrate/20201221173705_add_defer_date_to_trainees.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDeferDateToTrainees < ActiveRecord::Migration[6.0]
+  def change
+    add_column :trainees, :defer_date, :date
```

Comment:
We're storing the `withdraw_date` as a Datetime. Do we also want to keep the time information on deferrals? I might 'defer to' @jhenderson on this one.

Reply:
I might have made a mistake using `datetime`. I say go with `date` and I'll fix `withdraw_date` in the UUID PR.